src/content/docs/ko/guides/routing.mdx

```diff
@@ -298,6 +298,68 @@ if (!isLoggedIn(cookie)) {
 </html>
 ```
 
+## 리라이트
+
+<p><Since v="4.13.0" /></p>
+
+리라이트를 사용하면 브라우저를 다른 페이지로 리디렉션하지 않고도 다른 경로를 제공할 수 있습니다. 브라우저는 URL 표시줄에 원래 주소를 표시하지만, 대신 [`Astro.rewrite()`](/ko/reference/api-reference/#astrorewrite)에 제공된 URL의 콘텐츠를 표시합니다.
+
+:::tip
+영구적으로 이동한 콘텐츠를 제공하거나 사용자를 새 URL을 사용하는 다른 페이지로 안내하려면 (예: 로그인 후 사용자 대시보드) [리디렉션](#리디렉션)을 사용하세요.
+:::
+
+리라이트는 두 개의 다른 소스 파일을 유지 관리할 필요 없이 여러 경로 (예: `/products/shoes/men/` 및 `/products/men/shoes`/)에서 동일한 콘텐츠를 표시하는 데 유용할 수 있습니다.
```

Comment:
```suggestion
리라이트는 두 개의 다른 소스 파일을 유지 관리할 필요 없이 여러 경로 (예: `/products/shoes/men/` 및 `/products/men/shoes/`)에서 동일한 콘텐츠를 표시하는 데 유용할 수 있습니다.
```

Fixed the path typo in https://github.com/withastro/docs/pull/8993

Reply:
Thank you!